### PR TITLE
POL-66  Ensured all required perms are documented for each policy

### PR DESCRIPTION
**Pull request by Poojakravi**
POL-66  Added Permission block

### Description

We went through with all policies and found some policies without permissions mentioned. 

### Issues Resolved

Added permission block for both .pt and README file.

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: operational/fnms/schedule_fnms_reports/README.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ This policy has the following input parameters required when launching the polic
 ### Policy Actions
 No actions
 
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline operational/fnms/schedule_fnms_reports/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/azure/reserved_instances/utilization/README.md [hunk 2]
```diff
@@ -27,6 +27,16 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/utilization/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/azure/reserved_instances/recommendations/README.md [hunk 3]
```diff
@@ -29,6 +29,16 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/azure/reserved_instances/recommendations/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/azure/instances_log_analitics_utilization/README.md [hunk 4]
```diff
@@ -39,6 +39,17 @@ This policy has the following input parameters required when launching the polic
 - *Action Tag Key:Value* - The tag key:value pair to set on an instance that is underutilized.
 - *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analitics_utilization/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/aws/s3_bucket_size/README.md [hunk 5]
```diff
@@ -17,7 +17,13 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
-### Required Permissions
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/aws/s3_bucket_size/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/utilization/README.md [hunk 6]
```diff
@@ -19,6 +19,12 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/aws/reserved_instances/utilization/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/report_by_bc/README.md [hunk 7]
```diff
@@ -21,6 +21,12 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/aws/reserved_instances/report_by_bc/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/recommendations/README.md [hunk 8]
```diff
@@ -33,6 +33,34 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/aws/reserved_instances/recommendations/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/expiration/README.md [hunk 9]
```diff
@@ -18,6 +18,13 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/aws/reserved_instances/expiration/README.md]: for consistency please change to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/coverage/README.md [hunk 10]
```diff
@@ -18,6 +18,31 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```
Comment (cdwilhelm) [inline cost/aws/reserved_instances/coverage/README.md]: for consistency please change to Cloud Management - credential_viewer

File: compliance/azure/azure_long_stopped_instances/README.md [hunk 11]
```diff
@@ -16,6 +16,12 @@ The policy leverages the Azure API to check all instances that have been stopped
 ### Required RightScale Roles
 
 - policy_manager
+- credential_viewer
```
Comment (cdwilhelm) [inline compliance/azure/azure_long_stopped_instances/README.md]: for consistency please change to Cloud Management - credential_viewer

File: compliance/azure/ahub_manual/README.md [hunk 12]
```diff
@@ -21,6 +21,12 @@ Each license is good for one VM with up to 16 cores or two VM's with up to 8 cor
 - policy_designer
 - policy_manager
 - policy_publisher
+- credential_viewer
```
Comment (cdwilhelm) [inline compliance/azure/ahub_manual/README.md]: for consistency please change to Cloud Management - credential_viewer

File: operational/fnms/schedule_fnms_reports/README.md (hunk 1 above)
Comment (Poojakravi) [reply operational/fnms/schedule_fnms_reports/README.md]: Changed to Cloud Management - credential_viewer

File: cost/azure/reserved_instances/utilization/README.md (hunk 2 above)
Comment (Poojakravi) [reply cost/azure/reserved_instances/utilization/README.md]: Changed to Cloud Management - credential_viewer

File: cost/azure/reserved_instances/recommendations/README.md (hunk 3 above)
Comment (Poojakravi) [reply cost/azure/reserved_instances/recommendations/README.md]: Changed to Cloud Management - credential_viewer

File: cost/azure/instances_log_analitics_utilization/README.md (hunk 4 above)
Comment (Poojakravi) [reply cost/azure/instances_log_analitics_utilization/README.md]: Changed to Cloud Management - credential_viewer

File: cost/aws/s3_bucket_size/README.md (hunk 5 above)
Comment (Poojakravi) [reply cost/aws/s3_bucket_size/README.md]: Changed to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/utilization/README.md (hunk 6 above)
Comment (Poojakravi) [reply cost/aws/reserved_instances/utilization/README.md]: Changed to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/report_by_bc/README.md (hunk 7 above)
Comment (Poojakravi) [reply cost/aws/reserved_instances/report_by_bc/README.md]: Changed to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/recommendations/README.md (hunk 8 above)
Comment (Poojakravi) [reply cost/aws/reserved_instances/recommendations/README.md]: Changed to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/expiration/README.md (hunk 9 above)
Comment (Poojakravi) [reply cost/aws/reserved_instances/expiration/README.md]: Changed to Cloud Management - credential_viewer

File: cost/aws/reserved_instances/coverage/README.md (hunk 10 above)
Comment (Poojakravi) [reply cost/aws/reserved_instances/coverage/README.md]: Changed to Cloud Management - credential_viewer

File: compliance/azure/azure_long_stopped_instances/README.md (hunk 11 above)
Comment (Poojakravi) [reply compliance/azure/azure_long_stopped_instances/README.md]: Changed to Cloud Management - credential_viewer

File: compliance/azure/ahub_manual/README.md (hunk 12 above)
Comment (Poojakravi) [reply compliance/azure/ahub_manual/README.md]: Changed to Cloud Management - credential_viewer

**Pull request merged**